pytorch_lightning/callbacks/__init__.py

```diff
@@ -45,7 +45,6 @@
     "ModelPruning",
     "ModelSummary",
     "BasePredictionWriter",
-    "ProgressBar",
```

Comment:
I bet the next idea someone will have is to rename ProgressBarBase to ProgressBar 🤣